Library/Formula/yara.rb

```diff
@@ -35,6 +35,11 @@ def install
                           "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
     system "make", "install"
+    # build yara-python
+    system "cd", "yara-python"
+    system "python", "setup.py", "build"
+    system "python", "setup.py", "install"
+    system "cd", ".."
```

Comment:
You can replace all of this with:

```ruby
cd "yara-python" do
  system "python", *Language::Python.setup_install(prefix)
end
```

If yara-python has Python dependencies, they need to be captured as `resource` blocks and installed by the formula as well.